src/components/chip/themes/shared/chip.common.scss

```diff
@@ -45,4 +41,9 @@ $theme: $material;
     ::slotted(igc-circular-progress) {
         --diameter: #{sizable(rem(14px), rem(18px), rem(24px))};
     }
+
+    igc-icon,
+    ::slotted(igc-icon) {
+        --size: #{sizable(rem(18px), rem(18px), rem(24px))};
+    }
```

Comment:
As far as I remember the size of the icon in the chip should not be modified by the chip component. Rather, it should follow the default icon sizes.

Reply:
Correct, the issue is in the UI Kit.